pages/common/popd.md

```diff
@@ -8,10 +8,14 @@
 
 `popd`
 
-- Remove the Nth directory (starting from zero to the left from the list printed with `dirs`):
+- Remove the Nth directory (from the left of the list printed with `dirs`), couting starts at zero:
```

Comment:
```suggestion
- Remove the Nth directory (starting from zero to the left from the list printed with `dirs`):
```
I would suggest reverting this change as the old description seems more concise.